### DISCUSSION TITLE
Helm chart repo is abandoned

### DISCUSSION THREAD

**Discussion by iavael** [mastodon/mastodon#25789] [General]
Hi! Looks like helm chart repo https://github.com/mastodon/chart is abandoned by maintainers and PRs with important fixes are unreviewed.
Maybe maintainership of this repo should be designated to more k8s users of mastodon because there's definitely some demand for this helm chart.

**Comment by dunn** [comment]
fwiw there is also now a Bitnami chart available: https://github.com/bitnami/charts/tree/main/bitnami/mastodon

**Comment by timetinytim** [comment]
Since this is a pretty small team, there unfortunately wasn't enough manpower to give a lot of dedicated attention to the helm chart. I was recently brought on to help with devops/infra needs, as well as maintaining stuff like the helm chart, so there will be much more engagement and active development. Sincere apologies for all the silence and delays!
